tests/conftest.py

```diff
@@ -483,3 +483,26 @@ def zip_image_path(image_path, tmp_path_factory):
         f.write(image_path, arcname=os.path.basename(image_path))
         f.write(image_path, arcname=os.path.basename(image_path).replace(".jpg", "2.jpg"))
     return path
+
+
+@pytest.fixture(scope="session")
+def data_dir(tmp_path_factory):
```

Comment:
(nit) maybe better to just name it `data_dir_with_hidden_files` to make it more explicit